Xamarin.Forms.Platform.Android/FormsApplicationActivity.cs

```diff
@@ -193,6 +193,22 @@ protected override void OnStop()
 			OnStateChanged();
 		}
 
+		bool CanGoBack()
+		{
+			var backPressed = BackPressed;
```

Comment:
Why do you need a local variable? Can't you use `BackPressed` directly?

Reply:
I can and if it is needed I will change it.

Reply:
In general, I avoid allocating memory for local variables if I can directly use an existing one. In this scenario, I'm not sure what the compiler does in terms of optimization.

Reply:
@adrianknight89 This local variable will be released in the end of method with the help of garbage collector.